src/main/java/com/onwelo/tinvalidator/validator/LVValidator.java

```diff
@@ -0,0 +1,26 @@
+package com.onwelo.tinvalidator.validator;
+
+import java.util.regex.Pattern;
+
+public class LVValidator implements Validator {
+
+    private static final Pattern PATTERN = Pattern.compile("[0-59]\\d[01]\\d{3}[0-2]\\d{4}");
```

Comment:
Wydaje mi się że tutaj powinno wystarczyć [0-9]{11}
Bo o ile dla osób takich żyjących (czyli przypadek gdy `C1 < 4`) mamy potem ograniczone zakersy cyfr, to dla osób prawnych (czyli przypadek gdy `C1 > 3`) mogą już być dowolne cyfry.